internal/pkg/processor/config/config.go

```diff
@@ -13,45 +13,11 @@ import (
 	"github.com/go-playground/validator/v10"
 
 	"github.com/keboola/processor-split-table/internal/pkg/kbc"
+	slicerConfig "github.com/keboola/processor-split-table/internal/pkg/slicer/config"
```

Comment:
Od configu procesora oddelime to, co sa da pouzit aj v CLI, prakticky teda vsetko co je v `parameters`.